crates/teloxide-core/src/types/callback_query.rs

```diff
@@ -40,8 +40,8 @@ pub struct CallbackQuery {
     /// [games]: https://core.telegram.org/bots/api#games
     pub chat_instance: String,
 
-    /// A data associated with the callback button. Be aware that a bad client
-    /// can send arbitrary data in this field.
+    /// A data associated with the callback button. Be aware that the message
+    /// originated the query can contain no callback buttons with this data.
```

Comment:
It looks like the second sentence is outdated and is simply not true anymore. Could you please delete it altogether?

Reply:
Ok